test/plugins/collections.test.js

```diff
@@ -267,6 +302,22 @@ describe('collection plugin test', () => {
             });
         });
 
+        it('returns a collection only with pipelines that exist', () => {
```

Comment:
The `GET` operation filters as well? Does it save the changes to the collection?